packages/hardhat-etherscan/src/ABITypes.ts

```diff
@@ -19,10 +19,10 @@ export function isABIArgumentLengthError(
 ): error is ABIArgumentLengthError {
   return (
     error.code === "INVALID_ARGUMENT" &&
-    error.count &&
+    Boolean(error.count) &&
```

Comment:
Are these changes needed?

Reply:
This is just Franco V fixing up linter errors in [this commit](https://github.com/NomicFoundation/hardhat/pull/3546/commits/ae0b12e92a76bf978bd804b983a0458750b0a5e3) as discussed [here](https://nomicfoundation.slack.com/archives/C03UN893ZT4/p1674118392149789?thread_ts=1674034616.578069&cid=C03UN893ZT4)